### PR TITLE
Nebula as submodule

### PR DESCRIPTION
**Pull request by Warefoxe**
as Cargo does not support gitLFS for git dependencies I have moved Nebula to git submodules

### CODE REVIEW

**Comment by ospfranco** [comment]
Submodules might bring some troubles as one needs to pull them separately every time. Did you try adding a git fetch lfs code into the `build.rs`? Is that a possible way?

File: src-tauri/Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ log = { version = "^0.4" }
 tauri-plugin-log = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v1", features = [
   "colored",
 ] }
-nebula = { git = "https://github.com/nchapman/nebula.git", default-features = false, features = [
```
Comment (ospfranco) [inline src-tauri/Cargo.toml:20]: Could you add a package.json command to update the nebula submodule?

File: src-tauri/tauri.linux.conf.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "tauri": {
+    "allowlist": {
+      "fs": {
+        "scope": {
+          "requireLiteralLeadingDot": false
```
[the remark this answers is not in the document]
Comment (Warefoxe) [reply src-tauri/tauri.linux.conf.json:6]: I test only for Debian and Ubuntu, but according to Tauri docs I thinks so

File: src-tauri/Cargo.toml (hunk 1 above)
Comment (Warefoxe) [reply src-tauri/Cargo.toml:20]: Sorry, not understand.
updating submodules is required only for updating to the new version of Nebula, it is not regular build command.

really changes `git clone ...` command, now it should be `git clone --recurse-submodules ...`

Comment (Warefoxe) [reply src-tauri/Cargo.toml:20]: updated Readme.md

**Review by ospfranco — APPROVED** [review]

**Pull request merged**